lib/pengine/utils.c

```diff
@@ -159,8 +159,23 @@ pe__node_list2table(GList *list)
     return result;
 }
 
+/*!
+ * \internal
+ * \brief Compare two nodes by uname, with numeric portions sorted numerically
+ *
+ * Sort two node names case-insensitively like strcasecmp(), but with any
+ * numeric portions of the name sorted numerically. For example, "node10" will
+ * sort higher than "node9" but lower than "remotenode9".
+ *
+ * \param[in] a  First node to compare (must not be \c NULL)
+ * \param[in] b  Second node to compare (must not be \c NULL)
+ *
+ * \retval -1 \c a comes before \c b
+ * \retval  0 \c a and \c b are equal
+ * \retval  1 \c a comes after \c b
+ */
 gint
-sort_node_uname(gconstpointer a, gconstpointer b)
+pe__sort_node_uname(gconstpointer a, gconstpointer b)
```

Comment:
Ah I forgot to mention ... I've started using a consistent naming convention for this type of function.

* Use cmp in the name if it is a strcmp-like function (i.e. comparing two items and returning +/0/-)
* Use sort in the name if it sorts a list of items

Also, we should avoid using "uname" to refer to node names, since a node name can be different from the local uname.

Reply:
That fits my preferences too. "sort" is counterintuitive unless you're shuffling items around.

Reply:
done